msvc/projects/vc2015/jemalloc/jemalloc.vcxproj.filters

```diff
@@ -67,6 +67,9 @@
     <ClCompile Include="..\..\..\..\src\hpa.c">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\..\..\src\hpa_central.c">
```

Comment:
I think we also need to add`hpa_utils.c` in? Besides, I notice `thread_event_registry.c` is also missing from the filters files, can you help add it in as well?